### ISSUE TITLE
AttributeError: 'list' object has no attribute 'replace' - bug

### ISSUE THREAD

**Issue by JarbasAI** [siznax/wptools#64]
having this error randomly on some search terms, possible duplicate of #47
```
Elon_Musk (en)
{
  lang: en
  title: Elon_Musk
}
en.wikipedia.org (query) Elon_Musk
en.wikipedia.org (parse) 909036
www.wikidata.org (wikidata) Q317521
www.wikidata.org (claims) Q16|Q30|Q5|Q14590866|Q258
Traceback (most recent call last):
  File "/home/user/mycroft-core/mycroft/knowledge/testing_ground.py", line 9, in <module>
    musk = wptools.page('Elon Musk').get()#_claims()
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 615, in get
    self.get_wikidata(show, proxy, timeout)
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 752, in get_wikidata
    self._request('wikidata', show, proxy, timeout)
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 385, in _request
    self.get_imageinfo(False)
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 644, in get_imageinfo
    self._request('imageinfo', show, proxy, timeout)
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 359, in _request
    query = self._query(action, _fetch)
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 342, in _query
    files = self.__get_image_files()
  File "/usr/local/lib/python2.7/dist-packages/wptools/core.py", line 132, in __get_image_files
    fname = item.replace('_', ' ')
AttributeError: 'list' object has no attribute 'replace'
```

**Comment by siznax** [comment]
Thanks for reporting, @JarbasAI. I'm not getting this error with `0.2.1`.

Are you up to date?

```python
>>> p = wptools.page('Elon Musk').get(show=False)
Elon_Musk (en)
{
  lang: en
  title: Elon_Musk
}
en.wikipedia.org (query) Elon_Musk
en.wikipedia.org (parse) 909036
www.wikidata.org (wikidata) Q317521
www.wikidata.org (claims) Q16|Q30|Q5|Q14590866|Q258
en.wikipedia.org (imageinfo) File:Elon Musk 2015.jpg|File:Elon Musk - Th...

>>> wptools.__version__
'0.2.1'
```

**Comment by siznax** [comment]
Okay, I see @JarbasAI. The fix for #47 didn't make it into `0.2.1`. I pushed a new revision, `0.2.2` that includes that fix. Sorry for the confusion.
